specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json

```diff
@@ -0,0 +1,2640 @@
+        "tags": {
+          "description": "Contains resource tags defined as key/value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets the SKU of the resource."
+        },
+        "kind": {
+          "$ref": "#/definitions/Kind",
+          "description": "Required. Gets or sets the Kind of the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Entity Tag"
+        }
+      }
+    },
+    "Sku": {
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/SkuName",
+          "description": "The sku name"
+        },
+        "tier": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the sku tier. This is based on the SKU name.",
+          "enum": [
+            "Free",
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "The SKU of the cognitive services account."
+    },
+    "SkuName": {
+      "type": "string",
+      "description": "The name of SKU.",
+      "enum": [
+        "F0",
+        "S1"
+      ],
+      "x-ms-enum": {
+        "name": "SkuName",
+        "modelAsString": true
+      }
+    },
+    "Kind": {
+      "type": "string",
+      "description": "Indicates the type of bot service",
+      "enum": [
+        "sdk",
+        "designer",
+        "bot",
+        "function"
+      ],
+      "x-ms-enum": {
+        "name": "Kind",
+        "modelAsString": true
+      }
+    },
+    "Bot": {
+      "type": "object",
+      "description": "Bot resource definition",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BotProperties",
+          "description": "The set of properties specific to bot resource"
+        }
+      }
+    },
+    "BotProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The Name of the bot"
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the bot"
+        },
+        "iconUrl": {
+          "type": "string",
+          "description": "The Icon Url of the bot"
+        },
+        "endpoint": {
+          "type": "string",
+          "description": "The bot's endpoint"
+        },
+        "endpointVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The bot's endpoint version"
+        },
+        "msaAppId": {
+          "type": "string",
+          "description": "Microsoft App Id for the bot"
+        },
+        "configuredChannels": {
```

Comment:
so, this cannot be patchable, as it is readonly, this highlights the fact that your Patch bodies need to contain what can actually be patched